### ISSUE TITLE
Player controls, seekbar broken in v1.0.18+

### ISSUE THREAD

**Issue by Ur0s** [strawberrymusicplayer/strawberry#1283]
- [x] I have checked the [FAQ](https://wiki.strawberrymusicplayer.org/wiki/FAQ) for answers.
- [x] I have checked the [Changelog](https://github.com/strawberrymusicplayer/strawberry/blob/master/Changelog) that the issue is not already fixed. 
- [x] I believe this issue is a bug, and not a general technical issue, question or feature requests that can be discussed on the [forum](https://forum.strawberrymusicplayer.org/). 

**Describe the bug**
After upgrading from v1.0.17, controls in player / tray are broken.
Play button works to start/pause track, but it won't change icon to "pause" when track is playing.
Stop button is always disabled.
Previous / next track buttons work.
Seekbar doesn't work.

**To Reproduce**
Happens every time in v1.0.18+

**Expected behavior**
Player controls / seekbar should be working.

**Screenshots:**
![strawberry_1 0 20_controls](https://github.com/strawberrymusicplayer/strawberry/assets/2535667/a66cb9ea-def0-44fb-b257-d8079837ffcb)
![strawberry_1 0 20_tray](https://github.com/strawberrymusicplayer/strawberry/assets/2535667/815e0e9c-6310-423b-968c-1787acf366d0)
[strawberry-1.0.20.log](https://github.com/strawberrymusicplayer/strawberry/files/12777026/strawberry-1.0.20.log)

**System Information:**
 - Operating system: Linux
 - Strawberry Version: v1.0.20

**Additional context**
I've tried removing configuration / cache, issue still persists.

**Comment by jonaski** [comment]
No issues here with Qt 5.15.10 or 6.5.3.
I don't think this is a bug in strawberry.  Either some change in Qt that broke binary compatibility, or some patch applied to Qt that broke it.
According to your log:
```
11:44:26.011 WARN  unknown                          QObject::connect: Cannot queue arguments of type 'State' 
11:44:26.011 WARN  unknown                          (Make sure 'State' is registered using qRegisterMetaType().) 
```
This type is registered here: https://github.com/strawberrymusicplayer/strawberry/blob/0e44b10eecbe7943e7620a4f9a1c7edc0b2fe201/src/core/metatypes.cpp#L106

Try a rebuild of Strawberry.

**Comment by Ur0s** [comment]
Rebuilt v1.0.20 again, the issue mentioned is still present.

Rebuild v1.0.17 and everything works as expected with qt-core and all other related libraries unchanged:
![strawberry_v1 0 17_controls](https://github.com/strawberrymusicplayer/strawberry/assets/2535667/cb861592-a54d-4d89-bc60-557341a3171d)

**Comment by jonaski** [comment]
Did you try to clear ccache before building? `ccache -C`

**Comment by Ur0s** [comment]
ccache is not used.

**Comment by jonaski** [comment]
I booted the Gentoo liveusb (https://distfiles.gentoo.org/releases/amd64/autobuilds/20230924T163139Z/livegui-amd64-20230924T163139Z.iso), built 1.0.20 from source, with no issues, play/stop/pause/seeking works.
I also tried both version 1.0.18 and 1.0.20 from the Gentoo repository, no issues with those either.
Looking at the source code, the error you get does not make sense, since the "State" enum class is registered according to the source code.
This seem like a problem with your build or Qt installation, or mixing incompatible binaries. I suggest to seek support from your distro.
